ocaml/xapi/quicktest_import_raw_vdi.ml

```diff
@@ -0,0 +1,56 @@
+(*
+ * Copyright (C) 2017 Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+
+open Client
+open Quicktest_common
+open Http
+
+let http request f =
```

Comment:
Ooops, sorry, one more optional thing, although this could even be done in a subsequent refactoring PR: I assume this is the same function as the one in `quicktest_vdi_copy.ml`, so we should move it into `quicktest_common.ml`.